src/components/atoms/SfArrow/SfArrow.md

```diff
@@ -0,0 +1,19 @@
+# SfComponent
```

Comment:
could you please document it and move this part of docs fully to storybook? https://github.com/DivanteLtd/storefront-ui/pull/133 - for rel